src/Extension.ml

```diff
@@ -119,6 +119,7 @@ let activate (extension : Vscode.ExtensionContext.t) =
     (Vscode.Commands.register ~command:selectSandboxCommandId
        ~handler:(selectSandbox instance));
   Vscode.ExtensionContext.subscribe extension (Instance.disposable instance);
+  Vscode.ExtensionContext.subscribe extension (TaskProviderDune.create ());
```

Comment:
I feel this should be part of `Instance`. Similarly to how `DuneFormatter` is currently.